docs/_guide/styles.md

```diff
@@ -162,327 +122,653 @@ class MyElement extends LitElement {
+See [CSS Inheritance on MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/inheritance) for more information.
 
-_index.html_ 
+You can use CSS inheritance to set styles on an ancestor element that are inherited by its descendents:
 
 ```html
 <style>
-  my-element { 
-    font-family: Roboto;
-    font-size: 20;
-    color: blue;
-  }
+html { 
+  font-family: Roboto;
+}
 </style>
-...
-<my-element></my-element>
+<div>
+  <p>Uses Roboto</p>
+</div>
 ```
 
-An element type selector has higher specificity than the `:host` pseudo-class selector. 
-
-Styles set for a host element from outside its shadow DOM will override styles set with the `:host` or `:host()` pseudo-class selectors inside shadow DOM. For example:
+Similarly, host elements pass down inheritable CSS properties to their shadow trees. 
 
-_index.html_ 
+You can use the host element's type selector to style it:
 
 ```html
 <style>
-  /* Overrides the `color` property of the `:host` styles in my-element.js */
-  my-element { 
-    color: blue;
-  }
+  my-element { font-family: Roboto; }
 </style>
-...
 <my-element></my-element>
 ```
 
-_my-element.js_
+```js
+class MyElement extends LitElement {
+  render() { 
+    return html`<p>Uses Roboto</p>`; 
+  }
+}
+```
+
+{% include project.html folder="style/inherited3" openFile="index.html" %}
+
+You can also use the `:host` CSS pseudo-class to style the host from inside its own template:
+
+```js
+static get styles() {
+  return css`
+    :host {
+      font-family: Roboto;
+    }
+  `;
+}
+render() {
+  return html`
+    <p>Uses Roboto</p>
+  `;
+}
+```
+
+{% include project.html folder="style/inherited" openFile="my-element.js" %}
+
+{:.alert .alert-info}
+<div id="specificity">
+
+**Type selectors have higher specificity than :host.**
+
+An element type selector has higher specificity than the `:host` pseudo-class selector. Styles set for a custom element tag will override styles set with `:host` and `:host()`:
 
 ```html
 <style>
-  :host {
-    color: red;
-    background-color: aliceblue;
-  }
+  my-element { font-family: Courier; }
 </style>
+<my-element></my-element>
 ```
 
-### Custom CSS Properties {#css-properties}
+```js
+class MyElement extends LitElement {
+  static get styles() { 
+    return css`:host { font-family: Roboto; }`
+  }
+  render() {
+    return html`<p>Will use courier</p>`;
+  }
+}
+```
 
-Custom properties inherit down the DOM tree, and through shadow root boundaries. You can use this to let your users apply styles and themes to your elements.
+{% include project.html folder="style/specificity" openFile="index.html" %}
 
-For example, the following element sets its background color to a CSS variable that uses the value of the custom property `--myBackground` if it is available, and uses `yellow` otherwise:
+</div>
 
-_my-element.js_
+Any properties that a host element inherits will also be inherited by the elements in its template:
```

Comment:
Suggest removing this line and example, it seems like it's already been explained above?

Reply:
👍